examples/browser/wdio.conf.js

```diff
@@ -5,9 +5,14 @@ const path = require('path');
 const port = 3000;
 const host = 'localhost';
 
+const debug = process.env.DEBUG;
+
 let server;
 
 exports.config = {
+    debug: true,
+
+    execArgv: ['--inspect=127.0.0.1:5859'],
```

Comment:
why do you need it?

Reply:
The doc says it runs on 5859 when you add debug but no matter what I did it doesn't seem to work attaching to that port unless I add this argument (stated in the article I linked too, but not explained why...). I think it's maybe a bug in wdio?